src/asf_tools/hydrosar/flood_map.py

```diff
@@ -57,21 +57,22 @@ def get_waterbody(input_info: dict, threshold: Optional[float] = None) -> np.arr
     return water_array > threshold
 
 
-def iterative(hand: np.array, extent: np.array, x0: float = 7.5, water_levels: np.array = range(15),
-              minimization_metric: str = 'fmi'):
+def iterative(hand: np.array, extent: np.array, water_levels: np.array = np.arange(15),
```

Comment:
Do we want to leave it as 15 or 6 hard-coded?

Reply:
Since Minjeong didn't seem firmly committed to 6, and it is likely physically possible to have 15 m of water flooding an area, I'd prefer to leave it at 15 since it is an optional parameter at the function, cli, and HyP3 interfaces and can be specified at runtime.